Tasks/Vadeika/Task3.py

```diff
@@ -0,0 +1,115 @@
+"""Task3."""
+
+# User input. Validity check input.
+while True:
+    user_int = input("Enter the number of characters per line from 36 to 50:")
+    if not user_int.isdigit():
+        print("Incorrect input!")
+        continue
+    user_enter = int(user_int)
+    if user_enter > 50:
+        print("Incorrect input!")
+        continue
+    elif user_enter < 36:
+        print("Incorrect input!")
+        continue
+    else:
+        print(f'Successfully! You entered a number:{user_enter}')
+    break
+
+# Reading the file.
+with open(r'D:\WORK\PY\Python_less\TEMP\temp_text.txt', 'r', encoding='utf-8') as f:
+    file = f.read().split(' ')
```

Comment:
reading the whole file in memory is not the best approach, since it can be very huge, -10 points fir not optimal choice